common/views/components/ButtonSolid/ButtonSolid.tsx

```diff
@@ -133,10 +140,18 @@ export const SolidButton = styled(BaseButton).attrs<SolidButtonProps>(
   })
 )<SolidButtonProps>`
   background: ${props =>
-    props.theme.color(props.isDangerous ? 'red' : 'green')};
-  color: ${props => props.theme.color('white')};
+    props.theme.color(
```

Comment:
I don't like how verbose this gets. `defaultProps` looked promising e.g.

```
SolidButton.defaultProps = {
  colors: themeValues.buttonColours.default
}
```
but I couldn't get type inference to work if I did that, i.e. TS was still yelling at me that `props.colors` might be undefined.